components/PlantDetails/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Plant, UserPlants } from '@/types/schema';
+
+export default function PlantDetails({
+  detail,
```

Comment:
this should take in `setDate` and `setPlant`
See Kyle's [FilterDropdown ](https://github.com/calblueprint/trap-garden/pull/17/files#diff-b94a6601ff3864685038586da5aa156d32f6e12376888a97523ca15f312412a6) in `components/FilterDropdown.tsx` for an example of how to pass in setState react hooks